app/uk/gov/hmrc/estates/models/variation/VariationResponse.scala

```diff
@@ -67,7 +68,7 @@ object VariationResponse {
     }
 
   private def failure(errorResponse: ErrorResponse) = {
-    Logger.error(s"$logPrefix ${errorResponse.message}")
+    logger.error(errorResponse.message)
```

Comment:
Maybe put some identifying text on this, so someone can find this line from the log?

Reply:
Done